directly passing msg into function

yizenlim opened the pull request:
directly passing msg into function

yizenlim committed:
inner msg

psvensson reviewed (changes requested):
Thanks for bumping the version as well, but I'd rather that you leave the urls intact, for repository, et.c, if that is OK :)

yizenlim committed:
changed urls back to psvensson

yizenlim commented:
Hi Peter, so sorry i was worried that npm packages might not find it ways when i installed, so I changed the reference when I forked and published. I have changed them back already, see if the latest commit works for you ! cheers :D

psvensson reviewed (approved):
Haha, no worries. Thanks!
